src/dappsExplorer/search/NoResults.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { Trans } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import InfoIcon from 'src/icons/InfoIcon'
+import Colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+interface Props {
+  searchTerm: string
+  testID?: string
+}
+
+function NoResults({ testID, searchTerm }: Props) {
+  return (
+    <View testID={`${testID}/NoResults`} style={styles.viewContainer}>
```

Comment:
I'd expect this if passed a specific testID:
```suggestion
    <View testID={testID} style={styles.viewContainer}>
```

Reply:
Fixed in 6b9882a368accea810f9c533e9082c7fba72f920!